### PR TITLE
API1600 StorageVolume

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[API1600 support for StorageVolume]

### Issues Resolved
[]

### Check List
- [ ] New functionality includes testing.
  - [x]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: oneview/resource_volume.go [hunk 1]
```diff
@@ -287,9 +291,8 @@ func resourceVolumeUpdate(d *schema.ResourceData, meta interface{}) error {
 	isPermanent := d.Get("is_permanent").(bool)
 	isShareable := d.Get("is_shareable").(bool)
 
-	properties := d.Get("properties").(*schema.Set).List()[0].(map[string]interface{})
 	volume.Name = d.Get("name").(string)
-	volume.ProvisioningTypeForUpdate = properties["provisioning_type"].(string)
+	volume.ProvisioningTypeForUpdate = d.Get("provisioning_type").(string)
```
Comment (AnushaY1916) [inline oneview/resource_volume.go:295]: "provisioning_type" is under Properties Struct, why are we accessing using d.Get here and I see no such change or documentation regarding this in example?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+# [v1.4.0] (ureleased)
```
[the remark this answers is not in the document]
Comment (nabhajit-ray) [reply CHANGELOG.md]: removed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+### Notes
+- This release supports API1600 minimally where we can use OneView API1600 with this SDK. No new fields are added/deleted to support API1600. Complete support will be done in next releases.
+- Support for API V1200 for image streamer resource deployment plan.
```
[the remark this answers is not in the document]
Comment (nabhajit-ray) [reply CHANGELOG.md]: Removed

File: oneview/resource_volume.go (hunk 1 above)
Comment (nabhajit-ray) [reply oneview/resource_volume.go:295]: provisioning_type also there in the volume struct. We can use it from there.

**Pull request merged**